tests/example-marathon-app-with-missing-healthchecks.json

```diff
@@ -0,0 +1,450 @@
+{
```

Comment:
There's a lot of stuff in this json that isn't relevant to the test. Can you pare it down to only the parts necessary?

Reply:
yes